app/src/features/apiClient/screens/environment/components/environmentView/EnvironmentView.tsx

```diff
@@ -65,12 +48,7 @@ export const EnvironmentView = () => {
               onSearchValueChange={setSearchValue}
               currentEnvironmentName={environmentName}
             />
-            <VariablesList
-              searchValue={searchValue}
-              variables={variables}
-              setVariables={handleSetVariables}
-              removeVariable={handleRemoveVariable}
-            />
+            <VariablesList searchValue={searchValue} />
```

Comment:
Variables setter functions were defined outside this component because this list component is also being used in collection view. both env and collection variables have different variable updation logic.